phases/repo.go

```diff
@@ -109,27 +112,11 @@ func schemeToSSH(URL *url.URL) *url.URL {
 	return sshURL
 }
 
-func getProvider(hostName string) string {
-	hostParts := strings.Split(hostName, ".")
-	if len(hostParts) < 2 {
-		return "other"
-	}
-
-	if hostParts[len(hostParts)-1] == "com" && hostParts[len(hostParts)-2] == "github" {
-		return "github"
-	} else if hostParts[len(hostParts)-1] == "com" && hostParts[len(hostParts)-2] == "gitlab" {
-		return "gitlab"
-	} else if hostParts[len(hostParts)-1] == "org" && hostParts[len(hostParts)-2] == "bitbucket" {
-		return "bitbucket"
-	}
-	return "other"
-}
-
 func validateRepositoryAvailablePublic(url string) error {
 	if _, err := git.Clone(memory.NewStorage(), nil, &git.CloneOptions{
 		Auth:              nil,
 		URL:               url,
-		Progress:          os.Stdout,
+		Progress:          bytes.NewBuffer([]byte{}),
```

Comment:
Would add a progress indicator (https://github.com/bitrise-io/go-utils/tree/master/progress) while validation is running. The output could be printed out as a Debug level log.